### PR TITLE
feat: [PL-30425]: Audit Filter V2 API

### PR DESCRIPTION
**Pull request by reetika04**
## Describe your changes
This PR contains Spec First Audit V2 API. 
Additional fields added: List support for static filters

Testing:
Tested on the local environment 
Existing Audit listing should work with the V2 api.
On providing multiple StaticAuditFilter's, the api should return the data based on that.
`"static_filter": ["ENTITY_CHANGE_EVENTS", "LOGIN_EVENTS"]`

Case 1:
No static filter should return all the audit events
Case 2:
 Static filter with LOGIN_EVENTS should return all the login events
Case 3:
 Static filter with ENTITY_CHANGE_EVENTS should return all the entity change events
 Case 4:
 Static filter with RUNTIME_EVENTS should return all the pipeline run time events
 Case 5:
 Static filter with SYSTEM_EVENTS should return all the events which are done by the system principal 
 

## Checklist
- [ ] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>

- Feature build: `trigger feature-build`
- Immutable delegate `trigger publish-delegate`
</details>

<details>
  <summary>PR Check triggers</summary>

You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ti0, ti1`

- Compile: `trigger compile`
- CodeformatCheckstyle: `trigger checkstylecodeformat`
    - CodeFormat: `trigger codeformat`
    - Checkstyle: `trigger checkstyle`
- MessageMetadata: `trigger messagecheck`
- File-Permission-Check: `trigger checkpermission`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- PMD: `trigger pmd`
- Copyright Check: `trigger copyrightcheck`
- Feature Name Check: `trigger featurenamecheck`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`
- CodeFormatCheckstyle: `trigger checkstylecodeformat`
- SonarScan: `trigger ss`
- GitLeaks: `trigger gitleaks`
- Trigger all Checks: `trigger smartchecks`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/43047)

### CODE REVIEW

File: platform-service/modules/audit-service/contracts/openapi/v1/openapi.yaml [hunk 1]
```diff
@@ -155,6 +155,30 @@ paths:
         - x-api-key: []
       parameters:
         - $ref: '#/components/parameters/AccountHeader'
+  /v2/audits:
```
Comment (mankrit-singh) [inline platform-service/modules/audit-service/contracts/openapi/v1/openapi.yaml:158]: please dont use "v2/audits", we are standardizing all spec-first APIs having "v1" in prefix
suggestion - "v1/audits"

File: platform-service/modules/audit-service/contracts/openapi/v1/openapi.yaml [hunk 2]
```diff
@@ -377,6 +401,276 @@ components:
...
       description: Details of the last streamed batch
       x-internal: true
+    ResourceScopeDTO:
+      title: ResourceScopeDTO
+      x-stoplight:
+        id: 4pz290lmuwtl3
+      type: object
+      description: This has scope details for the resource defined in Harness.
+      properties:
+        accountIdentifier:
```
Comment (mankrit-singh) [inline platform-service/modules/audit-service/contracts/openapi/v1/openapi.yaml:411]: rename to account_identifier and similarly, please use snake case over camel case wherever applied

File: platform-service/modules/audit-service/contracts/openapi/v1/openapi.yaml [hunk 3]
```diff
@@ -539,6 +833,27 @@ components:
           schema:
             type: integer
           description: Maximum page size in Paginated response.
+    AuditListResponse:
```
Comment (mankrit-singh) [inline platform-service/modules/audit-service/contracts/openapi/v1/openapi.yaml:836]: please add example for the response if this is a public endpoint

**Review by reetika04 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/43047)* status: 7 file(s) to review. 3. unresolved discussions. 0 of 2 approvals obtained. 

___
*[`platform-service/modules/audit-service/contracts/openapi/v1/openapi.yaml` line 158 at r1](https://reviewable.io/reviews/harness/harness-core/43047#-NOryvGgAOUeMJf9uASq-r1-158:-NPFjIzK1Zp273a5c3t0:b-896fix) ([raw file](https://github.com/harness/harness-core/blob/d2f39db7134661f2c9b94a33306b8ff83b3b4246/platform-service/modules/audit-service/contracts/openapi/v1/openapi.yaml#L158)):*
<details><summary><i>Previously, mankrit-singh (Mankrit Singh) wrote…</i></summary><blockquote>

please dont use "v2/audits", we are standardizing all spec-first APIs having "v1" in prefix
suggestion - "v1/audits"
</blockquote></details>

Done.
___
*[`platform-service/modules/audit-service/contracts/openapi/v1/openapi.yaml` line 411 at r1](https://reviewable.io/reviews/harness/harness-core/43047#-NOryvGgAOUeMJf9uASq-r1-411:-NPFjKz46Yd_35Wyozv_:b-896fix) ([raw file](https://github.com/harness/harness-core/blob/d2f39db7134661f2c9b94a33306b8ff83b3b4246/platform-service/modules/audit-service/contracts/openapi/v1/openapi.yaml#L411)):*
<details><summary><i>Previously, mankrit-singh (Mankrit Singh) wrote…</i></summary><blockquote>

rename to account_identifier and similarly, please use snake case over camel case wherever applied
</blockquote></details>

Done.
___
*[`platform-service/modules/audit-service/contracts/openapi/v1/openapi.yaml` line 836 at r1](https://reviewable.io/reviews/harness/harness-core/43047#-NOryvGgAOUeMJf9uASq-r1-836:-NPFjJRp82Sfc5R6wNS3:b-896fix) ([raw file](https://github.com/harness/harness-core/blob/d2f39db7134661f2c9b94a33306b8ff83b3b4246/platform-service/modules/audit-service/contracts/openapi/v1/openapi.yaml#L836)):*
<details><summary><i>Previously, mankrit-singh (Mankrit Singh) wrote…</i></summary><blockquote>

please add example for the response if this is a public endpoint
</blockquote></details>

Done.

**Review by nishantsaini20 — CHANGES_REQUESTED** [review]
> Reviewed 1 of 7 files at r1, 2 of 8 files at r3, 1 of 1 files at r4, 3 of 6 files at r5, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/43047)* status: 7 file(s) to review. 1 change requests. 3. unresolved discussions. 0 of 2 approvals obtained. 

___
*[`platform-service/modules/audit-service/src/main/java/io/harness/audit/api/impl/AuditServiceImpl.java` line 141 at r5](https://reviewable.io/reviews/harness/harness-core/43047#-NPMSTLA6pVMXQT-AARM:-NPMSTLB0q8-BHxErQ5j:bvyedmi) ([raw file](https://github.com/harness/harness-core/blob/1ea0427b6b996a01ef2ef20ab249404020464346/platform-service/modules/audit-service/src/main/java/io/harness/audit/api/impl/AuditServiceImpl.java#L141)):*
> ```Java
>   @Override
>   public Page<AuditEvent> listV2(
>       String accountIdentifier, Pageable pageRequest, AuditFilterPropertiesV1DTO auditFilterPropertiesDTO) {
> ```

Instead of introducing listV2, we should convert AuditFilterPropertiesV1DTO to AuditFilterPropertiesDTO and use existing list method.

**Review by reetika04 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/43047)* status: 7 file(s) to review. 1 change requests. 4. unresolved discussions. 0 of 2 approvals obtained. 

___
*[`platform-service/modules/audit-service/src/main/java/io/harness/audit/api/impl/AuditServiceImpl.java` line 141 at r5](https://reviewable.io/reviews/harness/harness-core/43047#-NPMSTLA6pVMXQT-AARM:-NPMb6qRAra99pBPUl3C:bv1hzo8) ([raw file](https://github.com/harness/harness-core/blob/1ea0427b6b996a01ef2ef20ab249404020464346/platform-service/modules/audit-service/src/main/java/io/harness/audit/api/impl/AuditServiceImpl.java#L141)):*
<details><summary><i>Previously, nishantsaini20 (Nishant Saini) wrote…</i></summary><blockquote>

Instead of introducing listV2, we should convert AuditFilterPropertiesV1DTO to AuditFilterPropertiesDTO and use existing list method.
</blockquote></details>

Since they aren't compatible we cannot convert AuditFilterPropertiesV1DTO  to AuditFilterPropertiesDTO

**Review by nishantsaini20 — CHANGES_REQUESTED** [review]
> Reviewed 2 of 8 files at r3, 1 of 6 files at r5, 6 of 7 files at r6, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/43047)* status: 8 file(s) to review. 1 change requests. 4. unresolved discussions. 0 of 2 approvals obtained. 

___
*[`platform-service/modules/audit-service/contracts/src/main/java/io/harness/audit/beans/AuditFilterPropertiesDTO.java` line 58 at r6](https://reviewable.io/reviews/harness/harness-core/43047#-NPRYrJT0FkP97-h-T9I:-NPRYrJT0FkP97-h-T9J:b2nw3yu) ([raw file](https://github.com/harness/harness-core/blob/698e24df54f79a487dc1478cf97c2bcceb97438e/platform-service/modules/audit-service/contracts/src/main/java/io/harness/audit/beans/AuditFilterPropertiesDTO.java#L58)):*
> ```Java
>   @Schema(description = "Pre-defined Filter") StaticAuditFilter staticFilter;
> 
>   @Schema(description = "Pre-defined list of Filters") @Hidden List<StaticAuditFilter> staticFilters;
> ```

Check if this annotation is working as expected
___
*[`platform-service/modules/audit-service/src/main/java/io/harness/audit/api/impl/AuditServiceImpl.java` line 195 at r6](https://reviewable.io/reviews/harness/harness-core/43047#-NPRaLCcA_2AGhvYush9:-NPRaLCcA_2AGhvYushA:bf6re89) ([raw file](https://github.com/harness/harness-core/blob/698e24df54f79a487dc1478cf97c2bcceb97438e/platform-service/modules/audit-service/src/main/java/io/harness/audit/api/impl/AuditServiceImpl.java#L195)):*
> ```Java
>             Criteria.where(AuditEventKeys.action).is(UNSUCCESSFUL_LOGIN));
>     List<Criteria> systemCriteria = List.of(Criteria.where(AuditEventKeys.PRINCIPAL_TYPE_KEY).is(PrincipalType.SYSTEM));
>     if (staticFilter.contains(StaticAuditFilter.EXCLUDE_LOGIN_EVENTS)) {
> ```

The filters were suppose to be inclusive. The change seems to be missing.
___
*[`platform-service/modules/audit-service/src/main/java/io/harness/audit/mapper/ApiUtils.java` line 104 at r6](https://reviewable.io/reviews/harness/harness-core/43047#-NPR_ZcWAnIwy2aYhei4:-NPR_ZcWAnIwy2aYhei5:bexg7wh) ([raw file](https://github.com/harness/harness-core/blob/698e24df54f79a487dc1478cf97c2bcceb97438e/platform-service/modules/audit-service/src/main/java/io/harness/audit/mapper/ApiUtils.java#L104)):*
> ```Java
>   }
> 
>   public AuditFilterPropertiesDTO getAuditFilterPropertiesV2DTO(
> ```

Rename method to what its returning
___
*[`platform-service/modules/audit-service/src/main/java/io/harness/audit/mapper/ApiUtils.java` line 105 at r6](https://reviewable.io/reviews/harness/harness-core/43047#-NPR_lxV2-VriYqnuOiY:-NPR_lxV2-VriYqnuOiZ:b-q4qfoh) ([raw file](https://github.com/harness/harness-core/blob/698e24df54f79a487dc1478cf97c2bcceb97438e/platform-service/modules/audit-service/src/main/java/io/harness/audit/mapper/ApiUtils.java#L105)):*
> ```Java
> 
>   public AuditFilterPropertiesDTO getAuditFilterPropertiesV2DTO(
>       io.harness.spec.server.audit.v1.model.AuditFilterPropertiesV1DTO auditFilterPropertiesV2DTO) {
> ```

Rename variable name. Type is V1 where as variable name says V2

**Review by reetika04 — COMMENTED** [review]
> Reviewed 2 of 8 files at r3, 1 of 1 files at r4, 1 of 6 files at r5, 7 of 7 files at r8, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/43047)* status: 8 file(s) to review. 1 change requests. 7. unresolved discussions. 0 of 2 approvals obtained. 

___
*[`platform-service/modules/audit-service/src/main/java/io/harness/audit/api/impl/AuditServiceImpl.java` line 195 at r6](https://reviewable.io/reviews/harness/harness-core/43047#-NPRaLCcA_2AGhvYush9:-NPVoyJN6PAdkfvhL855:bbr6bpj) ([raw file](https://github.com/harness/harness-core/blob/698e24df54f79a487dc1478cf97c2bcceb97438e/platform-service/modules/audit-service/src/main/java/io/harness/audit/api/impl/AuditServiceImpl.java#L195)):*
<details><summary><i>Previously, nishantsaini20 (Nishant Saini) wrote…</i></summary><blockquote>

The filters were suppose to be inclusive. The change seems to be missing.
</blockquote></details>

The filters are inclusive
___
*[`platform-service/modules/audit-service/src/main/java/io/harness/audit/mapper/ApiUtils.java` line 104 at r6](https://reviewable.io/reviews/harness/harness-core/43047#-NPR_ZcWAnIwy2aYhei4:-NPVp43qC24zltVsByky:b-896fix) ([raw file](https://github.com/harness/harness-core/blob/698e24df54f79a487dc1478cf97c2bcceb97438e/platform-service/modules/audit-service/src/main/java/io/harness/audit/mapper/ApiUtils.java#L104)):*
<details><summary><i>Previously, nishantsaini20 (Nishant Saini) wrote…</i></summary><blockquote>

Rename method to what its returning
</blockquote></details>

Done.
___
*[`platform-service/modules/audit-service/src/main/java/io/harness/audit/mapper/ApiUtils.java` line 105 at r6](https://reviewable.io/reviews/harness/harness-core/43047#-NPR_lxV2-VriYqnuOiY:-NPVp4b_4stzSykhflaX:b-896fix) ([raw file](https://github.com/harness/harness-core/blob/698e24df54f79a487dc1478cf97c2bcceb97438e/platform-service/modules/audit-service/src/main/java/io/harness/audit/mapper/ApiUtils.java#L105)):*
<details><summary><i>Previously, nishantsaini20 (Nishant Saini) wrote…</i></summary><blockquote>

Rename variable name. Type is V1 where as variable name says V2
</blockquote></details>

Done.

**Review by nishantsaini20 — CHANGES_REQUESTED** [review]
> Reviewed 3 of 7 files at r8, 5 of 5 files at r9, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/43047)* status: 9 file(s) to review. 1 change requests. 5. unresolved discussions. 0 of 2 approvals obtained. 

___
*[`platform-service/modules/audit-service/src/main/java/io/harness/audit/api/impl/AuditServiceImpl.java` line 217 at r9](https://reviewable.io/reviews/harness/harness-core/43047#-NPWY3MnDHvy5UEsa-JD:-NPWY3MnDHvy5UEsa-JE:bte5m8h) ([raw file](https://github.com/harness/harness-core/blob/a1036d3b0da61c2b4e254b7bc25f8375d3e9ca08/platform-service/modules/audit-service/src/main/java/io/harness/audit/api/impl/AuditServiceImpl.java#L217)):*
> ```Java
>     List<Criteria> loginCriteria = List.of(Criteria.where(AuditEventKeys.action).in(loginEvents));
>     List<Criteria> systemCriteria = List.of(Criteria.where(AuditEventKeys.PRINCIPAL_TYPE_KEY).is(PrincipalType.SYSTEM));
>     List<Criteria> entityChangeEventCriteria = List.of(Criteria.where(AuditEventKeys.action).in(entityChangeEvents));
> ```

Why each of this criteria is list?
___
*[`platform-service/modules/audit-service/src/main/java/io/harness/audit/api/impl/AuditServiceImpl.java` line 228 at r9](https://reviewable.io/reviews/harness/harness-core/43047#-NPWYHPh9PWzIuQ4-nCM:-NPWYHPh9PWzIuQ4-nCN:bh0emkz) ([raw file](https://github.com/harness/harness-core/blob/a1036d3b0da61c2b4e254b7bc25f8375d3e9ca08/platform-service/modules/audit-service/src/main/java/io/harness/audit/api/impl/AuditServiceImpl.java#L228)):*
> ```Java
>       criteriaList.addAll(entityChangeEventCriteria);
>     }
>     return new Criteria().andOperator(criteriaList.toArray(new Criteria[0]));
> ```

Shouldn't it be orOperator?

**Review by reetika04 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/43047)* status: 9 file(s) to review. 1 change requests. 7. unresolved discussions. 0 of 2 approvals obtained. 

___
*[`platform-service/modules/audit-service/src/main/java/io/harness/audit/api/impl/AuditServiceImpl.java` line 217 at r9](https://reviewable.io/reviews/harness/harness-core/43047#-NPWY3MnDHvy5UEsa-JD:-NPWd7-I16xmDIhgoioh:b-896fix) ([raw file](https://github.com/harness/harness-core/blob/a1036d3b0da61c2b4e254b7bc25f8375d3e9ca08/platform-service/modules/audit-service/src/main/java/io/harness/audit/api/impl/AuditServiceImpl.java#L217)):*
<details><summary><i>Previously, nishantsaini20 (Nishant Saini) wrote…</i></summary><blockquote>

Why each of this criteria is list?
</blockquote></details>

Done.
___
*[`platform-service/modules/audit-service/src/main/java/io/harness/audit/api/impl/AuditServiceImpl.java` line 228 at r9](https://reviewable.io/reviews/harness/harness-core/43047#-NPWYHPh9PWzIuQ4-nCM:-NPWd7OZ5mRF6kIKFuEK:b-896fix) ([raw file](https://github.com/harness/harness-core/blob/a1036d3b0da61c2b4e254b7bc25f8375d3e9ca08/platform-service/modules/audit-service/src/main/java/io/harness/audit/api/impl/AuditServiceImpl.java#L228)):*
<details><summary><i>Previously, nishantsaini20 (Nishant Saini) wrote…</i></summary><blockquote>

Shouldn't it be orOperator?
</blockquote></details>

Done.

**Review by nishantsaini20 — CHANGES_REQUESTED** [review]
> Reviewed 1 of 2 files at r10, 4 of 4 files at r11, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/43047)* status: 7 file(s) to review. 1 change requests. 5. unresolved discussions. 0 of 2 approvals obtained. 

___
*[`platform-service/modules/audit-service/src/main/java/io/harness/audit/mapper/ApiUtils.java` line 81 at r11](https://reviewable.io/reviews/harness/harness-core/43047#-NPf3WzJ1mscnBXNXYIT:-NPf3WzJ1mscnBXNXYIU:b-n82rud) ([raw file](https://github.com/harness/harness-core/blob/09da0b83fcbb134c4b13dd77c180c03733224d7a/platform-service/modules/audit-service/src/main/java/io/harness/audit/mapper/ApiUtils.java#L81)):*
> ```Java
>       io.harness.spec.server.audit.v1.model.StaticAuditFilter staticAuditFilter) {
>     try {
>       return StaticAuditFilterV2.valueOf(staticAuditFilter.toString());
> ```

Any reason for changing this from name to toString()?

**Review by nishantsaini20 — COMMENTED** [review]
> Reviewed 5 of 5 files at r13, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/43047)* status: 7 file(s) to review. 1 change requests. 5. unresolved discussions. 0 of 2 approvals obtained.

**Review by reetika04 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/43047)* status: 7 file(s) to review. 1 change requests. 6. unresolved discussions. 0 of 2 approvals obtained. 

___
*[`platform-service/modules/audit-service/src/main/java/io/harness/audit/mapper/ApiUtils.java` line 81 at r11](https://reviewable.io/reviews/harness/harness-core/43047#-NPf3WzJ1mscnBXNXYIT:-NPpMLs4DN-sXJIL6mDb:b-896fix) ([raw file](https://github.com/harness/harness-core/blob/09da0b83fcbb134c4b13dd77c180c03733224d7a/platform-service/modules/audit-service/src/main/java/io/harness/audit/mapper/ApiUtils.java#L81)):*
<details><summary><i>Previously, nishantsaini20 (Nishant Saini) wrote…</i></summary><blockquote>

Any reason for changing this from name to toString()? 
</blockquote></details>

Done.

**Review by reetika04 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/43047)* status: 7 file(s) to review. 1 change requests. 6. unresolved discussions. 0 of 2 approvals obtained. 

___
*[`platform-service/modules/audit-service/src/main/java/io/harness/audit/api/impl/AuditServiceImpl.java` line 141 at r5](https://reviewable.io/reviews/harness/harness-core/43047#-NPMSTLA6pVMXQT-AARM:-NPpNFe26GjDH_fb4xpH:bb6un1j) ([raw file](https://github.com/harness/harness-core/blob/1ea0427b6b996a01ef2ef20ab249404020464346/platform-service/modules/audit-service/src/main/java/io/harness/audit/api/impl/AuditServiceImpl.java#L141)):*
<details><summary><i>Previously, reetika04 (Reetika) wrote…</i></summary><blockquote>

Since they aren't compatible we cannot convert AuditFilterPropertiesV1DTO  to AuditFilterPropertiesDTO 
</blockquote></details>

Done
___
*[`platform-service/modules/audit-service/src/main/java/io/harness/audit/api/impl/AuditServiceImpl.java` line 195 at r6](https://reviewable.io/reviews/harness/harness-core/43047#-NPRaLCcA_2AGhvYush9:-NPpNIEACvIHzch9Z4Jp:bb6un1j) ([raw file](https://github.com/harness/harness-core/blob/698e24df54f79a487dc1478cf97c2bcceb97438e/platform-service/modules/audit-service/src/main/java/io/harness/audit/api/impl/AuditServiceImpl.java#L195)):*
<details><summary><i>Previously, reetika04 (Reetika) wrote…</i></summary><blockquote>

The filters are inclusive
</blockquote></details>

Done

**Review by nishantsaini20 — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/43047)* status: 7 file(s) to review. 3. unresolved discussions. 1 of 2 approvals obtained.

File: platform-service/modules/audit-service/contracts/openapi/v1/openapi.yaml [hunk 4]
```diff
@@ -155,6 +155,31 @@ paths:
         - x-api-key: []
       parameters:
         - $ref: '#/components/parameters/AccountHeader'
+  /v1/audits:
+    post:
+      summary: ''
```
Comment (mankrit-singh) [inline platform-service/modules/audit-service/contracts/openapi/v1/openapi.yaml:160]: please add a one line summary

**Review by reetika04 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/43047)* status: 7 file(s) to review. 4. unresolved discussions. 0 of 2 approvals obtained. 

___
*[`platform-service/modules/audit-service/contracts/openapi/v1/openapi.yaml` line 160 at r13](https://reviewable.io/reviews/harness/harness-core/43047#-NOryvGgAOUeMJf9uASq-r13-160:-NPpeR777hV5G0LsFeDp:b-896fix) ([raw file](https://github.com/harness/harness-core/blob/7c129ea5a9955c1ccd911c18f7a6735d3a70487e/platform-service/modules/audit-service/contracts/openapi/v1/openapi.yaml#L160)):*
<details><summary><i>Previously, mankrit-singh (Mankrit Singh) wrote…</i></summary><blockquote>

please add a one line summary
</blockquote></details>

Done.

**Review by mankrit-singh — COMMENTED** [review]
> Reviewed 2 of 8 files at r3, 1 of 6 files at r5, 3 of 7 files at r8, 1 of 2 files at r10, 4 of 5 files at r13, 1 of 1 files at r14, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/43047)* status: 1 file(s) to review. 4. unresolved discussions. 0 of 2 approvals obtained.

**Review by mankrit-singh — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/43047)* status: 1 file(s) to review. 1 of 2 approvals obtained.

**Review by nishantsaini20 — APPROVED** [review]
> Reviewed 1 of 1 files at r14, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/43047)* status: :shipit: complete! ✓

**Pull request merged**